source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/FactSetRenderer.java

```diff
@@ -100,15 +108,19 @@ public View render(
 
             DateTimeParser parser = new DateTimeParser(fact.GetLanguage());
 
-            // Handle Title
             String titleWithFormattedDates = parser.GenerateString(fact.GetTitleForDateParsing());
-            factRow.addView(createTextView(context, RendererUtil.handleSpecialText(titleWithFormattedDates), hostConfig.GetFactSet().getTitle(), hostConfig, spacing, renderArgs.getContainerStyle()));
-
-            // Handle Value
             String valueWithFormattedDates = parser.GenerateString(fact.GetValueForDateParsing());
-            factRow.addView(createTextView(context, RendererUtil.handleSpecialText(valueWithFormattedDates), hostConfig.GetFactSet().getValue(), hostConfig, 0, renderArgs.getContainerStyle()));
+            if (titleWithFormattedDates.length() != 0 || valueWithFormattedDates.length() != 0)
```

Comment:
you can use the method titleWithFormattedDates.isEmpty() instead of length comparisson